mineral-ui/mineral-ui
# [radio,checkbox]: Add hideLabel prop

### Description

Adds a `hideLabel` prop to accessibly hide the label on Radios and Checkboxes

### Motivation and context

We needed this capability when developing selectable rows in the new Table component.  Agile Central needs this capability for use within their DataTable too.

### Screenshots, videos, or demo, if appropriate

https://choice-hide-label--mineral-ui.netlify.com/components/checkbox#hide-label

### How to test

1. Add the `hideLabel` prop to any of the examples in Checkbox, CheckboxGroup, Radio, RadioGroup
2. Ensure that the component size/layout does not shift
3. Try it with a screenreader and verify that the label is announced properly

### Types of changes

- New feature (non-breaking change which adds functionality)

**BREAKING CHANGES**: Rename theme variables `RadioControl_marginHorizontal`
to `RadioText_marginHorizontal` and `CheckboxControl_marginHorizontal` to
`CheckboxText_marginHorizontal`


### Checklist

* [x] Tested in [sandbox](https://github.com/facebookincubator/create-react-app#getting-started) if new component or breaking change - nope
